python/config.cfg

```diff
@@ -39,7 +39,7 @@ train_conv_mode = fft
 
 # cost function: square_loss, binomial_cross_entropy, softmax_loss, auto
 # auto mode will match the out_type: boundary-softmax_loss, affinity-binomial_cross_entropy
-cost_fn = auto
+cost_fn = square_loss
```

Comment:
@nicholasturner1 why do you change it to square loss? I think that auto is better for default setting.